src/views/components/comment-item/comment-item.js

```diff
@@ -7,25 +7,70 @@ import Moment from 'react-moment';
 import 'moment/locale/he';
 import 'moment-timezone';
 import Linkify from 'react-linkify';
-import { I18n } from 'react-i18next';
+import {I18n} from 'react-i18next';
+import Icon from "../icon";
+import ToolTip from "react-portal-tooltip";
+import Button from "../button";
+import Textarea from "react-textarea-autosize";
 
 export class CommentItem extends Component {
   constructor() {
     super(...arguments);
-    this.state = {};
+    this.state = {
+      body: this.props.comment.body,
+      isInEditMode: false
+    };
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.setState({body: this.props.comment.body});
+  }
```

Comment:
Some reading on this method (And how it's being replaced) - 
https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops